src/lib/Oscilloscope.svelte

```diff
@@ -4,7 +4,7 @@
  function generatePath (data: number[]) : string {
      const strs: string[] = [];
      data.forEach((d: number, i: number) => strs.push(
-         `${i / (waveData.length - 1)},${d}`
+         `${i / (waveData.length - 1)},${-d}`
```

Comment:
I'm interested in the rationale. Oscilloscopes were upside down?

Reply:
Is this why the graphs are upside down? @44100hertz

Reply:
The SVG coordinate space is that Y goes from top to bottom, which is the opposite of how audio is traditionally displayed.